<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>surfaceDialog</class>
 <widget class="QDialog" name="surfaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surfaces</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="controlGroup">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="surfaceLabel">
          <property name="text">
           <string>Surface:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="surfaceCombo">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nullColorLabel">
          <property name="text">
           <string>â– </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nullColorButton">
          <property name="text">
           <string>Select color</string>
          </property>
         </widget>
        </item>	   
       </layout>
      </item>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="minLabel">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="minEdit">
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="maxLabel">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="maxEdit">
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="keepLimits">
          <property name="text">
           <string>Keep limits</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="appearanceGroup">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="useNormals">
          <property name="text">
           <string>Compute normals</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="angleLabel">
          <property name="text">
           <string>Feature angle:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="featureAngle">
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="value">
           <number>45</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="opacityLabel">
          <property name="text">
           <string>Opacity:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="opacitySpin">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="clipPlane">
          <property name="text">
           <string>Use clip plane</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../ElmerGUI.qrc">
         <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../ElmerGUI.qrc">
         <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc"/>
  <include location="../ElmerGUI.qrc"/>
 </resources>
 <connections/>
</ui>
